File: src/main/java/net/lomeli/trophyslots/core/ModItems.java
package net.lomeli.trophyslots.core;

import net.minecraft.item.Item;

import cpw.mods.fml.common.registry.GameRegistry;

public class ModItems {
    public static Item trophy;

    public static void registerItems() {
        trophy = new ItemTrophy();
        GameRegistry.registerItem(trophy, "trophy");
    }
}

File: src/main/kotlin/net/lomeli/trophyslots/core/ModItems.kt
package net.lomeli.trophyslots.core

import net.minecraft.item.Item
import net.minecraftforge.fml.common.registry.GameRegistry

object ModItems {
    @JvmField
    var trophy: Item? = null

    fun registerItems() {
        trophy = ItemTrophy()
        registerItem(trophy!!, "trophy")
    }

    fun registerItem(item: Item, name: String) {
        item.setRegistryName(name)
        GameRegistry.register(item)
    }
}